api/models/user.py

```diff
@@ -1,18 +1,31 @@
+""" User Model """
+
+from werkzeug.security import generate_password_hash
 from api.models.store import store
+
+
 class user(object):
```

Comment:
Review this: https://www.python.org/dev/peps/pep-0008/#class-names

Reply:
Fixed